### DISCUSSION TITLE
Total Conversions

### DISCUSSION THREAD

**Discussion by Gorilli09** [SuperTux/supertux#2892] [Ideas]
I'm thinking about making a total conversion for the game and I don't feel like creating a fork. Can you support total conversions in the game?

**Comment by provigz** [comment]
What do you mean by "total conversion"?

**Reply by Gorilli09** [reply]
I meant a complete modification inside the game.

**Reply by provigz** [reply]
Unless you want to modify assets, it is not possible in-game, since you'd have to modify the source code, which is compiled to give you the game executable as a result.

You don't need to fork the repository to compile the game, you only need to clone it and compile the game appropriately.
